src/components/ArticlesList.jsx

```diff
@@ -22,7 +22,7 @@ const ArticlesList = () => {
     if(isLoading) return <p>Loading...</p>
     
     return (
-        <>
+        <section className="section__cards">
         {articles.map((article) =>  {
             console.log(article)
```

Comment:
remove console.log

Reply:
I noticed this just before our chat and removed it - thanks!